.gitignore

```diff
@@ -258,4 +269,7 @@ paket-files/
 
 # Python Tools for Visual Studio (PTVS)
 __pycache__/
-*.pyc
\ No newline at end of file
+*.pyc
+
+*.trx
```

Comment:
Add a comment above this describing what they are for? Or move them to the "Files created and used by our build scripts" section?